crt_portal/cts_forms/templates/forms/complaint_view/index/filters/incident_location_filter.html

```diff
@@ -1,18 +1,11 @@
-{% extends "forms/complaint_view/index/filter-no-dropdown.html" %}
+{% extends "forms/complaint_view/index/filter.html" %}
 
 {% block id %}incident-location{% endblock %}
+{% block label %}Organization name{% endblock %}
 {% block classes %}summary_input{% endblock %}
 
 {% block content %}
-{% if ENABLED_FEATURES.fuzzy_location_name %}
-<span class="usa-tooltip" data-position="right" data-classes="display-inline" title='Tip: Use "~some location" (without quotes) to match locations that sound or look similar to some location'>
-{% endif %}
-  <div id="incident_location-filter">
-    <label for="id_{{form.fields.location_name.widget.attrs.name}}">
-      {{ form.location_name }}
-    </label>
-  </div>
-{% if ENABLED_FEATURES.fuzzy_location_name %}
-</span>
-{% endif %}
+<div id="incident_location-filter">
```

Comment:
should we have some indication in the filter pills above the table that fuzziness is being used?

Reply:
hmm good point. To keep it simple, I might just add a suffix like "Organization name (fuzzy)" or "Organization name (exact)"

Reply:
decided not to keep it simple - here's what I've come up with:

![Image](https://github.com/usdoj-crt/crt-portal-management/assets/15126660/fde3808b-37e7-4b28-933b-94b6524e0a54)
![Image](https://github.com/usdoj-crt/crt-portal-management/assets/15126660/bf87ad67-c4c4-4ea4-a8ff-da3fbeebd5c0)
![Image](https://github.com/usdoj-crt/crt-portal-management/assets/15126660/50171027-4154-48f6-a490-84c80b06f14a)